### ISSUE TITLE
Featured Lockbox content is missing words for privacy and terms

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#3665]
https://testpilot.dev.mozaws.net/__version__

```js
{
  commit: "0aa1d837ac973412e576caa32065d5221d48d79c",
  version: "",
  source: "https://github.com/mozilla/testpilot",
  build: "https://circleci.com/gh/mozilla/testpilot/11761"
}
```

![firefox_test_pilot](https://user-images.githubusercontent.com/557895/42110575-7ead4a42-7b96-11e8-83d3-6aef7cec9394.png)

**Comment by devinreams** [comment]
Hmmmm, thanks @pdehaan.

I intentionally commented-out the language at 680bc3c3b4a6eeb31dc1f2b58afebbc73843ee37 because it wasn't actually having any affect but maybe I was doing something else wrong. I'll look at this again...

**Comment by devinreams** [comment]
Fully fixed at #3666, I believe 🤞
